.github/workflows/checkSite.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: Merge gob branch
+        run: git --work-tree=. checkout gob .
```

Comment:
This command doesn't work. It seems the following works.
```
git worktree add ./gob gob
cp -r gob/* .
```

Reply:
This command worked with `upstream/gob` in place of `gob`. Switched it to your command.